backend/src/controllers/validationController.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from "express";
+import User from "../models/User";
+import getErrorMessage from "../helpers/getErrorMessage";
+require("dotenv").config();
+
+const validationController = {
+  checkUsernameAvailability: async (req: Request, res: Response) => {
+    const { username } = req.query;
+    console.log(username);
```

Comment:
Te logi, gdzie nie errory ogolnie pousuwaj wszystkie, i jak sa errory to daj console.error zamiast console.log